main.yml

```diff
@@ -108,6 +108,17 @@ compliance:
   frontend:
     paths:
       - /rhel/compliance
+    sub_apps:
+      - id: ''
+        title: Reports
+        default: true
+        reload: compliance
```

Comment:
Has this been tested locally? I haven't seen a `reload` defined for a default app before. If it was tested and working, I'll go ahead and approve this

Reply:
Yeah, it works well - I removed it though and it works the same, to be fair I just took the `drift` app as an example and thought it was necessary.